test/validation/api/ipsec/ipsec.c

```diff
@@ -1111,14 +1119,17 @@ static odp_queue_t plain_queue_create(const char *name)
 	return odp_queue_create(name, NULL);
 }
 
-static odp_event_t sched_dest_queue_deq(void)
+static odp_event_t sched_dest_queue_deq(uint64_t wait)
 {
-	return odp_schedule(NULL, ODP_SCHED_NO_WAIT);
+	if (wait == 0)
+		return odp_schedule(NULL, ODP_SCHED_NO_WAIT);
+
+	return odp_schedule(NULL, wait);
```

Comment:
This should use odp_schedule_wait_time() to convert nanoseconds to a value suitable for the wait parameter.

Reply:
Ack.